packages/app/components/settings/tabs/account.tsx

```diff
@@ -29,13 +38,35 @@ export const AccountTab = ({ index = 0 }: AccountTabProps) => {
     []
   );
 
+  const user = useUser();
+
+  const { disconnectSpotify } = useDisconnectSpotify();
+  const { connectSpotify } = useConnectSpotify();
+  const isDark = useIsDarkMode();
+
   return (
     <SettingScrollComponent index={index}>
       <SettingAccountSlotHeader />
       {accountSettings?.length > 0 &&
         accountSettings.map((item) => (
           <AccountSettingItem {...item} key={item.id} />
         ))}
+      <View tw="mb-4 flex-row items-center px-4">
```

Comment:
if make this style is the same as `PushNotificationsTab` will be more consistent, wdyt? 
![image](https://user-images.githubusercontent.com/37520667/199535819-66ce52aa-23c7-4bcb-8c0f-328428d1c304.png)